tests/install-litmus_test.go

```diff
@@ -72,29 +73,58 @@ var _ = Describe("BDD of litmus installation", func() {
 
 		It("Should check for creation of Litmus", func() {
 
-			//Installing Litmus
-			By("Installing Litmus")
-			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-operator-latest.yaml").Run()
-			Expect(err).To(BeNil(), "Failed to install litmus")
+			//Installing Crds
+			By("Installing crds")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/chaos_crds.yaml").Run()
+			Expect(err).To(BeNil(), "Failed to install crds")
+			if err != nil {
```

Comment:
what is the need of this check if we already have a ` Expect(err).To(BeNil(), "Failed to install crds")` check